packages/caip/src/utils.test.ts

```diff
@@ -72,3 +77,31 @@ describe('getChainReferenceFromChainId', () => {
     expect(() => getChainReferenceFromChainId(invalidChainId)).toThrow()
   })
 })
+
+describe('chainIdToFeeAssetId', () => {
+  it('returns a chain fee assetId for a given ETH chainId', () => {
```

Comment:
We're not very consistent with this across codebases, but let's try to keep the separate vocabulary for ETH the asset vs. Ethereum the network
```suggestion
  it('returns a chain fee assetId for a given Ethereum chainId', () => {
```

Reply:
I like it. Good call